scripts/dockerfiles/rsyslog_conf/etc/logrotate.d/rsyslog

```diff
@@ -1,14 +1,13 @@
 /var/log/syslog
 {
 	rotate 7
-	size 5M
+	maxsize=10M
+	minsize=9M
```

Comment:
Any particular reason for the size change?
Not that rotating at 5M made sense before…

Also, you can actually have `minsize` == `maxsize` and it does the right thing.